js/pulse.js

```diff
@@ -282,7 +285,17 @@
                     }
                 }
 
+                // Check for some callbacks to expose plugin internals
```

Comment:
When a new embed_code (movie) is set, we run through this code again.  If the client doesn't want callbacks the second time around, it would be good to clear out all the old callbacks before parsing the new ones.